app.js

```diff
@@ -54,6 +54,7 @@ const health = healthcheck.configure({
 });
 app.get('/', health);
 app.get('/health', health);
+app.get('/health/liveness', health);
```

Comment:
Given that "healthcheck module" is already exposing both "/health" and "/health/liveness" route/path, it may be better to install the healthcheck using the middleware approach which will avoid the duplication of the path. An example can be seen in this project: https://github.com/hmcts/cnp-plum-frontend/blob/master/server.js